test/voting_to_manage_emission_funds_test.js

```diff
@@ -545,7 +545,8 @@ contract('VotingToManageEmissionFunds [all features]', function (accounts) {
       const emissionFundsBalanceOld = await web3.eth.getBalance(emissionFunds.address);
       const receiverBalanceOld = await web3.eth.getBalance(receiver);
 
-      await voting.setTime(VOTING_START_DATE);
+      const time = VOTING_START_DATE + minBallotDuration + 1;
+      await voting.setTime(time);
```

Comment:
Why did you change this to `VOTING_START_DATE + minBallotDuration + 1`? The vote must pass right beginning from `VOTING_START_DATE`. The voter shouldn't wait until `minBallotDuration` is expired.

Reply:
Is this because of `ballotInfo[4].should.be.equal(true); // isFinalized` ?

Reply:
> Is this because of `ballotInfo[4].should.be.equal(true); // isFinalized` ?

Yes, it is